automation/system_test/prepare.py

```diff
@@ -354,6 +376,34 @@ def _download_provctl(self):
         self._run_command("chmod", args=["+x", provctl])
         return provctl
 
+    def _wait_until_finished(
+        self,
+        command: str,
+        command_name_to_wait_for: str,
+        timeout: int = 600,
+        interval: int = 10,
+    ):
+        finished = False
+        counter = 0
+        while not finished and counter != timeout:
```

Comment:
```suggestion
        while not finished and counter < timeout:
```
I know there shouldn't be an issue, but just in case